spec/support/es_spec_helper.rb

```diff
@@ -56,6 +56,8 @@ def create_es_index(klass)
     create_es_index(Manifest)
   end
   config.after :suite do
-    stop_es_server
+    if es_server_running?
```

Comment:
ah yes - this as an accidental re-add. removed webmock to try and test the actual CDX API responses, which was causing es problems. removed at c2a6d24